### PR TITLE
Bench updates

### PR DESCRIPTION
**Pull request by RiskyMH**
Some fixes to bench scripts.

Adds the bench site's webhook to the buildkite script.

### CODE REVIEW

**Comment by robobun** [comment]
@RiskyMH, your commit 885b498b29d71deef1f7d4cc9bb5e58843269538 is building: [`#5748`](https://buildkite.com/bun/bun/builds/5748)

File: .buildkite/scripts/upload-release.sh [hunk 1]
```diff
@@ -206,6 +221,7 @@ function create_release() {
 
   update_github_release "$tag"
   create_sentry_release "$tag"
+  send_bench_webhook "$tag"
```
Comment (RiskyMH) [inline .buildkite/scripts/upload-release.sh:228]: @Electroid this is untested and I'm assuming things here, let me know if its wrongly done or if better way.

also this needs the `BENCHMARK_URL` to make it more hidden from public.

**Comment by Jarred-Sumner** [comment]
This is fine and we should merge it, but it shouldn't error if the benchmark_url is unset

**Comment by RiskyMH** [comment]
> This is fine and we should merge it, but it shouldn't error if the benchmark_url is unset

Done, I also updated the environment variable to `$BENCHMARK_URL` for clarity.

**Pull request merged**